dev/InfoBar/InfoBarPanel.cpp

```diff
@@ -117,10 +128,10 @@ winrt::Size InfoBarPanel::ArrangeOverride(winrt::Size const& finalSize)
                 auto const desiredSize = child.DesiredSize();
                 if (desiredSize.Width != 0 && desiredSize.Height != 0)
                 {
-                    auto horizontalMargin = winrt::InfoBarPanel::GetSpacingInHorizontalOrientation(child);
+                    auto horizontalMargin = winrt::InfoBarPanel::GetHorizontalOrientationMargin(child);
 
                     horizontalOffset += hasPreviousElement ? (float)horizontalMargin.Left : 0;
-                    child.Arrange(winrt::Rect{ horizontalOffset, (float)horizontalMargin.Top, desiredSize.Width, finalSize.Height });
+                    child.Arrange(winrt::Rect{ horizontalOffset, (float)horizontalOrientationPadding.Top + (float)horizontalMargin.Top, desiredSize.Width, finalSize.Height });
```

Comment:
Why do we use desiredSize/finalSize here, but desiredSize/desiredSize for vertical?

Reply:
Not sure.... I switched it to desiredSize for all and can't tell the difference, so hopefully it's OK.

Reply:
@teaP  can you test to ensure this does not regress behavior ?

Reply:
I did, that's what I mean by "can't tell the difference". :D

Reply:
I think finalSize.Height would be the height of the tallest child, so desiredSize.Height sounds right. That also matches StackPanel's code.

Reply:
Yep, agreed!

Reply:
(Actually, finalSize.Height should be the height of the tallest child plus the padding?)